src/content/docs/ecosystem/delegators/overview.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Delegators
+description: Overview of Delegations on SKALE
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+
+## Introduction
+
+The SKALE Token [SKL] is the first ERC token with Ethereum
+native delegation and staking functionality. SKL is a modified
+[ERC-777](https://eips.ethereum.org/EIPS/eip-777) token backward compatible with the [ERC-20](https://eips.ethereum.org/EIPS/eip-20) standard.
+
+<Aside type="note" title="Understanding the Backward Compatibility">
+  It's important to note that the backward compatible nature means that
+  the SKL token can be viewed on MetaMask, found on popular CEX + DEX, etc.
+  The ERC-777 functionality is built-on top of the core ERC-20 standard.
+</Aside>
+
+You may use any ERC-20 compatible wallet to hold and delegate your
+tokens for/into the SKALE Network. Transfers and approvals work the same
+in ERC-777 as in ERC-20 -- the only difference is that ERC-777
+allows for certain functions and checks before transfers are executed.
+The SKL token uses this pre-transfer function to conduct checks
+to ensure that tokens are not actively delegated.
+
+### Important Information
+
+- The SKALE Token Address on the Ethereum Mainnet is: [0x00c83aeCC790e8a4453e5dD3B0B4b3680501a7A7](https://etherscan.io/address/0x00c83aeCC790e8a4453e5dD3B0B4b3680501a7A7)
+- GitHub Contract Code can be found [here]()
```

Comment:
SKL contract code: https://github.com/skalenetwork/skale-manager/blob/b039ebebb16a1a2a97a707d7ba6a0f2636413f62/contracts/SkaleToken.sol

Reply:
Thank you

Reply:
Resolved by [578991a](https://github.com/skalenetwork/docs.skale.space/pull/127/commits/578991ae0b2a57b76bcaede88d85ad33dff249bf)